File: src - java/Esercizi/CodaPari/CodaPari_ric.java
	private boolean _codaPariRic(Record p) {
		if (p == null)
			return false;

		if (p.succ != null)
			return _codaPariRic(p.succ);
		else
			return p.el % 2 == 0;
	}
	
	public boolean codaPariRic() {
		return _codaPariRic(first);
	}

File: src - c++/Esercizi/CodaPari/CodaPari_ric.cpp
bool Lista::_CodaPari(const PRec p) const {
  if (p) {
    if (p->succ)
      return _CodaPari(p->succ);
    else
      return ((p->el % 2) == 0);
  } else
    return 0;   
}  

bool Lista::CodaPari() const {
  return _CodaPari(first);
}
